### ISSUE TITLE
USWDS file input component is incompatible with CSP

### ISSUE THREAD

**Issue by jeffpw-goog** [civiform/civiform#8143]
**Describe the bug**
The [file input component](https://designsystem.digital.gov/components/file-input/) uses `onerror` inline Javascript event handlers which are incompatible with CSP: https://github.com/uswds/uswds/blob/369cf3fdf9abf721a357e864a0f6adbf667ba862/packages/usa-file-input/src/index.js#L394-L427

**To Reproduce**

With CSP enabled:
1. Create a program with a file upload question
2. Turn on the northstar flag
3. On your local machine create a text file and rename it to *.jpg
4. Apply to the program and upload the file to the file upload question
5. Observe that the preview image is broken, and the browser console shows a CSP error `refused to execute inline event handler`

**Expected behavior**
The preview image should be a generic file icon, and there should be no CSP error

**Screenshots**

**Actual:**
![image](https://github.com/user-attachments/assets/0fb66813-ca83-4b9a-8d13-6105e066214b)

**Expected:**
![image](https://github.com/user-attachments/assets/6eedef2d-e2c8-4aba-be58-7f880280b006)

**Comment by jeffpw-goog** [comment]
I filed https://github.com/uswds/uswds/issues/5990 in the USWDS project.

**Comment by jeffpw-goog** [comment]
This is blocked on USWDS merging and releasing a fix to https://github.com/uswds/uswds/issues/5990. I proposed a fix https://github.com/uswds/uswds/pull/5997 for their consideration.

**Comment by jeffpw-goog** [comment]
To move forward with CSP while we wait for the USWDS fix, I updated our browser tests to expect the broken behavior: #8248.

**Comment by jeffpw-goog** [comment]
It turns out there are actually [two usages](https://github.com/search?q=repo%3Aciviform%2Fciviform%20usa-file-input&type=code) of the USWDS file input component:
* The northstar file upload question (as identified above, and is not live today)
* The program image upload page (is live today)

So both of those cases will have this issue.

**Comment by devin-civi** [comment]
The upcoming multi-file upload feature is also soft-impacted on this. It'd be really, really nice to have it, but not strictly necessary.

**Comment by jeffpw-goog** [comment]
The fix for the USWDS issue https://github.com/uswds/uswds/issues/5990 has been merged and will be released in version 3.10.0 which appears to be planned shortly or at least this month.

**Comment by jeffpw-goog** [comment]
USWDS 3.10.0 was released: https://github.com/uswds/uswds/releases/tag/v3.10.0

**Comment by jeffpw-goog** [comment]
Confirmed that the file upload widget now behaves as expected (with CSP which is now always enabled) on program image upload. 🎉
